RadarSDK/RadarAPIClient.h

```diff
@@ -52,6 +52,9 @@ typedef void (^_Nonnull RadarDistanceAPICompletionHandler)(RadarStatus status, N
 typedef void (^_Nonnull RadarMatrixAPICompletionHandler)(RadarStatus status, NSDictionary *_Nullable res, RadarRouteMatrix *_Nullable matrix);
 
 typedef void (^_Nonnull RadarConfigAPICompletionHandler)(RadarStatus status, RadarMeta *_Nullable meta);
+
+typedef void (^_Nonnull RadarSendEventsAPICompletionHandler)(RadarStatus status, NSDictionary *_Nullable res, NSArray<RadarEvent *> *_Nullable event);
```

Comment:
nit: my bad, this should probably be singular `RadarSendEventAPICompletionHandler`, as we're only supporting sending one event, even though the callback comes back with an array (since events from `/track` are included)

Reply:
Ok, I'll change it.

Reply:
Done.